26-register_registration/275-creator_navigates_to_files_and_license_tab.feature

```diff
@@ -63,9 +69,8 @@ Feature: Creator navigates to Files and License tab
     Given Creator adds a file
     When they see the file in the list of files
     Then they can see information about:
-      | Version          |
-      | Publish date     |

```

Comment:
publish date / embargo skal vel fortsatt vises for filer som skal publiseres

Reply:
Ja, på filer som har embargo må publiseringsdato vises